Maestro.json

```
@@ -267,6 +273,312 @@
+      "Runways": [
+        { "Identifier": "09" },
+        { "Identifier": "16" },
+        { "Identifier": "27" },
+        { "Identifier": "34" }
+      ],
+      "RunwayModes": [
+        {
+          "Identifier": "09",
+          "Runways": [
+            { "Identifier": "09", "DefaultLandingRateSeconds": 180 }
+          ]
+        },
+        {
+          "Identifier": "16",
+          "Runways": [
+            { "Identifier": "16", "DefaultLandingRateSeconds": 180 }
+          ]
+        },
+        {
+          "Identifier": "27",
+          "Runways": [
+            { "Identifier": "27", "DefaultLandingRateSeconds": 180 }
+          ]
+        },
+        {
+          "Identifier": "34",
+          "Runways": [
+            { "Identifier": "34", "DefaultLandingRateSeconds": 180 }
+          ]
+        },
+        {
+          "Identifier": "LAHSO",
+          "Runways": [
+            { "Identifier": "34", "DefaultLandingRateSeconds": 180 },
+            { "Identifier": "27", "DefaultLandingRateSeconds": 180 }
+          ]
+        },
+        {
+          "Identifier": "34/27",
+          "Runways": [
+            { "Identifier": "34", "DefaultLandingRateSeconds": 180 },
+            { "Identifier": "27", "DefaultLandingRateSeconds": 180 }
+          ]
+        },
+        {
+          "Identifier": "34/09",
+          "Runways": [
+            { "Identifier": "34", "DefaultLandingRateSeconds": 180 },
+            { "Identifier": "09", "DefaultLandingRateSeconds": 180 }
+          ]
+        },
+        {
+          "Identifier": "16/27",
+          "Runways": [
+            { "Identifier": "16", "DefaultLandingRateSeconds": 180 },
+            { "Identifier": "27", "DefaultLandingRateSeconds": 180 }
+          ]
+        },
+        {
+          "Identifier": "16/09",
+          "Runways": [
+            { "Identifier": "16", "DefaultLandingRateSeconds": 180 },
+            { "Identifier": "09", "DefaultLandingRateSeconds": 180 }
+          ]
+        }
+      ],
+      "Arrivals": [
+        {
+          "FeederFix": "ARBEY",
+          "ArrivalRegex": "/ARBE\\dA/",
+          "AircraftType": "Jet",
+          "RunwayIntervals": {
+            "09": 11,
+            "16": 9,
+            "27": 12,
+            "34": 13
+          }
+        },
+        {
+          "FeederFix": "ARBEY",
+          "ArrivalRegex": "/ARBE\\dA/",
+          "AircraftType": "NonJet",
+          "RunwayIntervals": {
+            "09": 12,
+            "16": 10,
+            "27": 14,
+            "34": 15
+          }
+        },
+        {
+          "FeederFix": "BOOIN",
+          "ArrivalRegex": "/BOIN\\dA/",
+          "AircraftType": "Jet",
+          "RunwayIntervals": {
+            "09": 15,
+            "16": 12,
+            "27": 10,
+            "34": 13
+          }
+        },
+        {
+            "FeederFix": "BOOIN/V",
+            "ArrivalRegex": "/BOIN\\dA/",
```

Ahh. I think I see what you're trying to do here.

I think this will achieve what you're looking for. This will match the `BOOIN` STAR with the `V` suffix:
```suggestion
            "FeederFix": "BOOIN",
            "ArrivalRegex": "/BOIN\\dV/",
```

---

again, missed that, fixed